### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_ScreenReader: Screenreader does not read the role of "Start Debugging" button correctly and also name is missing for "Add Configuration" combobox beside that.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52302]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA/JAWS is ON.
2. Navigate to Activity Bar and select "View Debug "(Cntrl+Shift+D) button.
3. Navigate to debugging section toolbar under heading "Debug".

**Actual:**
Screenreader does not read the role of "Start Debugging" button correctly. It reads it as "section".
Name is missing for "Add Configuration" combobox beside that. Name is read as the value that is selected for the combobox.

**Same issue has been observed with JAWS.**

**Expected:**
Screenreader should read the role correctly for "Start Debugging" button and name should be provided for "Add Configuration" combobox beside that.

**Recommendations:**
Provide proper name to the combobox and role to "Start Debugging" button.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the correct name for the combobox to select the appropriate value and also will not be able to identify button present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_ScreenReader_NameRoleMissing.pptx](https://github.com/Microsoft/vscode/files/2114697/A11y_VSCode_ViewDebug_ScreenReader_NameRoleMissing.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
dup of https://github.com/Microsoft/vscode/issues/52332
